jmdaemon/jmdaemon/daemon_protocol.py

```diff
@@ -940,14 +946,14 @@ def init_connections(self, nick):
         incomplete transaction is wiped.
         """
         self.jm_state = 0  #uninited
```

Comment:
Heh, i don't know if it's just me, but in my head 'uninited' meaning 'un-init-ed' or 'uninitialized' feels fine but, sure, I can believe that is not actually a word :) And it could be very confusing to read. So yeah.

Reply:
Just removed it for now, since it's not helpful. Done in f86ee530c95c2dadd23bfe4a0bc349ce11c408e6